planet/specs.py

```diff
@@ -17,11 +17,19 @@
 import os
 from pathlib import Path
 
-
 DATA_DIR = 'data'
 PRODUCT_BUNDLE_SPEC_NAME = 'orders_product_bundle_2020_03_10.json'
-SUPPORTED_TOOLS = ['band_math', 'clip', 'composite', 'coregister',
-                   'file_format', 'reproject', 'tile', 'toar', 'harmonize']
+SUPPORTED_TOOLS = [
+    'band_math',
+    'clip',
+    'composite',
+    'coregister',
+    'file_format',
+    'reproject',
+    'tile',
+    'toar',
+    'harmonize'
+]
```

Comment:
This change is because of the modifier. It goes away when the modifier is removed. I'm ambivalent to the change here.